server/user/tests.py

```diff
@@ -7,6 +8,11 @@ class UserTest(TestCase):
     """Test module for users APIs"""
 
     def setUp(self):
+        # Disable SSL redirect in tests
+        settings_manager = override_settings(SECURE_SSL_REDIRECT=False)
```

Comment:
Same as previous comments. Just to ensure this setting is. consistent everywhere even if we need to make changes in the future

Reply:
Moved to common class